### PR TITLE
Add emitChainEvents script.

### PR DESCRIPTION
**Pull request by jnaviask**
This will let people emit arbitrary chain events which are run through the event handlers by preparing a chain events json file (example provided). Should be useful for testing, I recommend not using in production though.

### CODE REVIEW

File: server/scripts/emitChainEvents.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      log.error(`Malformed event: ${JSON.stringify(event, null, 2)}`);
+      process.exit(1);
+    }
+  }
+
+  // load chain and handlers
+  let handlers: IEventHandler[];
+  try {
+    await sequelize.authenticate();
+    const node = await models.ChainNode.findOne({
```
Comment (maxwellfoley3) [inline server/scripts/emitChainEvents.ts:52]: should display appropriate error if ChainNode does not exist

Comment (maxwellfoley3) [reply server/scripts/emitChainEvents.ts:52]: ie wrong input for the chain namee

File: server/scripts/emails.ts [hunk 2]
```diff
@@ -173,7 +173,7 @@ export const sendImmediateNotificationEmail = async (subscription, emailObject)
   emailObject.bcc = 'raymond+bcc@commonwealth.im';
 
   try {
-    console.log('sending immediate notification email');
+    console.log(`sending immediate notification email to ${emailObject.to}`);
```
Comment (maxwellfoley3) [inline server/scripts/emails.ts:176]: unclear on the purpose of this, as this is undefined due to being initialized to null in createImmediateNotificationEmailObject

File: server/scripts/setupChainEventListeners.ts [hunk 3]
```diff
@@ -193,7 +198,7 @@ const setupChainEventListeners = async (
     });
     return [ node.chain, subscriber ];
   }));
-  return _.object<{ [chain: string]:  IEventSubscriber<any, any> }>(subscribers);
```
Comment (maxwellfoley3) [inline server/scripts/setupChainEventListeners.ts:196]: looks like accidental change

File: server/scripts/emails.ts (hunk 2 above)
Comment (jnaviask) [reply server/scripts/emails.ts:176]: ? It's set just a few lines up.

File: server/scripts/setupChainEventListeners.ts (hunk 3 above)
Comment (jnaviask) [reply server/scripts/setupChainEventListeners.ts:196]: It's a whitespace fix.

**Pull request merged**